### PR TITLE
Text Folder Cleaning

### PR DESCRIPTION
**Pull request by SpockBauru**
Cleaned shop items that now are in `RedirectedResources`
Replaced `JapToWesternNumbers.txt` with `CircledNumbers.txt` since now XUA have RegexPostProcessing
Added some texts that spams in `_AutoGeneratedTranslations.txt` like the alphabet when you enter an H-Scene

### CODE REVIEW

**Review by ManlyMarco — APPROVED** [review]

File: Translation/en/RedirectedResources/assets/abdata/action/list/shopinfo/30/30/translation.txt [hunk 1]
```diff
@@ -7,11 +7,11 @@
 Hな話題を聞く=Ask for an adult topic
 ちょっとアダルトな話題はいかがですか？=How about a topic that's a bit more adulty?
 媚薬香水=Aphrodisiac fragrance 
-香りを嗅ぐと気分がムラムラとしてくる特殊な香水。\n（翌日の一日間、女の子の行動に影響します）=A fragrance that boosts one's libido whenm smelled. \n (Affects the actions of a girl for the next day)
+香りを嗅ぐと気分がムラムラとしてくる特殊な香水。\n（翌日の一日間、女の子の行動に影響します）=A fragrance that boosts one's libido whenm smelled.\n(Affects the actions of a girl for the next day)
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/action/list/shopinfo/30/30/translation.txt:10]:
```suggestion
香りを嗅ぐと気分がムラムラとしてくる特殊な香水。\n（翌日の一日間、女の子の行動に影響します）=A fragrance that boosts one's libido when smelled.\n(Affects the actions of a girl for the next day)
```
or even just
```suggestion
香りを嗅ぐと気分がムラムラとしてくる特殊な香水。\n（翌日の一日間、女の子の行動に影響します）=A fragrance that boosts one's libido.\n(Affects the actions of a girl for the next day)
```

File: Translation/en/RedirectedResources/assets/abdata/action/list/shopinfo/30/30/translation.txt [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
 利尿剤=Diuretic drug
-飲むと頻繁にトイレに行きたくなる薬。\n（翌日の一日間、女の子の行動に影響します）=A drug that makes you want to go to the toilet.  \n (Affects the actions of a girl for the next day)
+飲むと頻繁にトイレに行きたくなる薬。\n（翌日の一日間、女の子の行動に影響します）=A drug that makes you want to go to the toilet.\n(Affects the actions of a girl for the next day)
 発汗剤=Diaporetic drug
-//飲むと発汗が促進され、頻繁にシャワーやお風呂に行きたくなる薬。\n（翌日の一日間、女の子の行動に影響します）=A drug that increases sweating, thus making you want to go to the shower or bath more often. \n (Affects the actions of a girl for the next day)
+飲むと発汗が促進され、頻繁にシャワーやお風呂に行きたくなる薬。\n（翌日の一日間、女の子の行動に影響します）=A drug that increases sweating, thus making you want to go to the shower or bath more often.\n(Affects the actions of a girl for the next day)
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/action/list/shopinfo/30/30/translation.txt:14]:
```suggestion
飲むと発汗が促進され、頻繁にシャワーやお風呂に行きたくなる薬。\n（翌日の一日間、女の子の行動に影響します）=A drug that increases sweating, thus making you want to bathe more often.\n(Affects the actions of a girl for the next day)
```

File: Translation/en/Text/Main/CircledNumbers.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+//Circled numbers, because they are hard to read
```
Comment (GeBo1) [inline Translation/en/Text/Main/CircledNumbers.txt:1]: Should be resolution scoped `width < 1920` to be consistent

File: Translation/en/Text/Main/H-Scenes.txt [hunk 4]
```diff
@@ -111,3 +111,8 @@
 飲んでね=Swallow it
 吐いてね=Spit it
 イキそう=I'm about to cum
+
+//---------------------------------------------------------
+// Unsorted: Things that I don't know where they came from
+//---------------------------------------------------------
+あいうえおかきくけこさ=あいうえおかきくけこさ
```
Comment (GeBo1) [inline Translation/en/Text/Main/H-Scenes.txt:118]: Add a comment that this is just placeholder text that doesn't need to be translated

**Review by GeBo1 — CHANGES_REQUESTED** [review]
Few minor things

File: Translation/en/RedirectedResources/assets/abdata/action/list/shopinfo/30/30/translation.txt (hunk 1 above)
Comment (SpockBauru) [reply Translation/en/RedirectedResources/assets/abdata/action/list/shopinfo/30/30/translation.txt:10]: I'll use the second one

File: Translation/en/Text/Main/CircledNumbers.txt (hunk 3 above)
Comment (SpockBauru) [reply Translation/en/Text/Main/CircledNumbers.txt:1]: I'll also make a proper header for this one

**Review by GeBo1 — APPROVED** [review]

**Pull request merged**